### ISSUE TITLE
Does a 24-word mnemonic correspond to multiple addresses, or is there a 1:1 relationship between them?

### ISSUE THREAD

**Issue by akegaviar** [akegaviar/ton-foo#157]
Does anyone know if multiple addresses correspond to a 24-word mnemonic or if this is a 1:1 relationship between them? On networks like Ethereum, a mnemonic can be used to generate multiple private keys.

This question was imported from Telegram Chat: [https://t.me/tondev_eng/2584](https://t.me/tondev_eng/2584)

## Answer

**Yes, it is possible for a 24-word mnemonic phrase to correspond to an unlimited number of addresses. **  ⭕️

This phrase is generated using a deterministic wallet seed and is used to derive multiple public and private key pairs following the BIP32 hierarchical deterministic wallet (HD wallet) standard.

**❗️It is important to note that the relationship between the mnemonic phrase and its corresponding key pairs is one-to-many, rather than many-to-many. **

In other words, each unique 24-word mnemonic phrase generates a unique set of key pairs. While it is possible to derive multiple public addresses from a single mnemonic phrase, each of those public addresses can be traced back to that same mnemonic phrase in a unique way.❤️‍🔥

---

[Original](https://answers.ton.org/question/1538718118329716736/does-a-24-word-mnemonic-correspond-to-multiple-addresses-or-is-there-a-1-1-relationship-between-them)
